lib/fs.js

```diff
@@ -476,7 +477,10 @@ function open(path, flags, mode, callback) {
   } else {
     mode = parseFileMode(mode, 'mode', 0o666);
   }
+
   const flagsNumber = stringToFlags(flags);
+  validateInt32(flagsNumber, flags);
```

Comment:
```suggestion
  validateInt32(flagsNumber, 'flags');
```
Shouldn't this be `'flags'`?